sample-emojisearch/app/build.gradle

```diff
@@ -0,0 +1,65 @@
+plugins {
+  id 'com.android.application'
+  id 'kotlin-android'
```

Comment:
I have lots of feedback around the Android build bits. Can I just push to your branch later? (Also thanks for fixing 2-space indent).

Reply:
YES PLEASE